tools/nemo_forced_aligner/README.md

```diff
@@ -0,0 +1,48 @@
+# NeMo Forced Aligner (NFA)
+
+A tool for doing Forced Alignment using Viterbi decoding of NeMo CTC-based models.
+
+## How do I use NeMo Forced Aligner?
+To use NFA, all you need to provide is a correct NeMo manifest (with 'audio_filepath' and 'text' fields).
+
+Call the align function in align.py, specifying the parameters as follows:
+
+* `model_name`: Any Quartznet, Citrinet, Conformer CTC model should work, in any language (only English has been tested so far). You can provide a pretrained model name or a path to a saved '.nemo' file.
+
+> Note: If you want to transcribe a long audio file (longer than ~5-10 mins), do not use Conformer CTC model as that will likely OOM.
+
+* `model_downsample_factor`: should be 2 if your model is QuartzNet, 4 if it is Conformer CTC, 8 if it is Citrinet.
+
+* `manifest_filepath`: The path to the manifest of the data you want to align.
```

Comment:
Add some note as to what the manifest should contain - audio_filepath, text and duration. Also note that for alignment, it is possible to pass in unsegmented data as a single manifest line, with the audio file + unsegmented text data.